### PR TITLE
Chore/cherry pick 16 changes

### PR DESCRIPTION
**Pull request by makelicious**

### CODE REVIEW

**Comment by Zangetsu101** [comment]
@makelicious this is ok to merge right?

**Comment by Zangetsu101** [comment]
Also these changes need to go to countryconfig :sweat_smile:

**Comment by rikukissa** [comment]
@makelicious @Zangetsu101 closing this for now. Please reopen if this was still going to merged

**Comment by Zangetsu101** [comment]
@makelicious This does need to be merged right? The current qa is actually running this branch I think

File: CHANGELOG.md [hunk 1]
```diff
@@ -51,6 +51,37 @@
...
+- Add query mapper for International Postal Code field
+- Add support for image compression configuration
+- Provide env variables for metabase admin credentials
+- Improved formatting of informant name for inProgress declaration emails
+- Rename `farajaland-map.geojson` to `map.geojson` to not tie implementations into example country naming
+- Remove `splitView` option from DOCUMENT_UPLOADER_WITH_OPTION field [#114](https://github.com/opencrvs/opencrvs-countryconfig/pull/114)
+- Enable authentication for certificates endpoint [#188](https://github.com/opencrvs/opencrvs-countryconfig/pull/188)
+
+## [1.4.1](https://github.com/opencrvs/opencrvs-countryconfig/compare/v1.4.0...v1.4.1)
+
```
Comment (Zangetsu101) [inline CHANGELOG.md]: This shouldn't be here. It should be merged with line 52 instead

**Comment by makelicious** [comment]
@Zangetsu101 these are already in countryconfig, too. I deployed this to qa last week, since the environment was broken. Depending on the process we have for updating `release-v1.6.0` branch, we can merge this or if you have some preferred way of doing it

**Comment by Zangetsu101** [comment]
I have just noticed a anomaly in the CHANGELOG. We can go ahead and merge this in after it's fixed

**Comment by makelicious** [comment]
@Zangetsu101 Thanks! Did I interpret it correctly that the extra title was the problem? Comparing to countryconfig the changes were actually related to 1.5

**Comment by Zangetsu101** [comment]
> @Zangetsu101 Thanks! Did I interpret it correctly that the extra title was the problem? Comparing to countryconfig the changes were actually related to 1.5

Yes that's the one

File: CHANGELOG.md [hunk 2]
```diff
@@ -49,7 +49,36 @@
...
+- Groom's and Bride's name, printIssue translation variables updated [#124](https://github.com/opencrvs/opencrvs-countryconfig/pull/124)
+- Add query mapper for International Postal Code field
+- Add support for image compression configuration
+- Provide env variables for metabase admin credentials
+- Improved formatting of informant name for inProgress declaration emails
+- Rename `farajaland-map.geojson` to `map.geojson` to not tie implementations into example country naming
+- Remove `splitView` option from DOCUMENT_UPLOADER_WITH_OPTION field [#114](https://github.com/opencrvs/opencrvs-countryconfig/pull/114)
+- Enable authentication for certificates endpoint [#188](https://github.com/opencrvs/opencrvs-countryconfig/pull/188)
+
+## [1.4.1](https://github.com/opencrvs/opencrvs-countryconfig/compare/v1.4.0...v1.4.1)
```
Comment (Zangetsu101) [inline CHANGELOG.md]:
```suggestion
## [1.4.1](https://github.com/opencrvs/opencrvs-farajaland/compare/v1.4.0...v1.4.1)
```

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**